runtime/queries/cue/highlights.scm

```diff
@@ -150,11 +152,15 @@
 ] @constant.builtin
 
 ; Interpolations
+(interpolation
+  "\\" @punctuation.special
+  (_)
+  ")" @punctuation.special) @none
 
-(interpolation "\\(" @punctuation.special (_) ")" @punctuation.special) @none
-
-(interpolation "\\(" (identifier) @variable ")")
+(interpolation
+  "\\"
+  (identifier) @variable
+  ")")
```

Comment:
I think you're missing some escapes in the code...

Reply:
Re-added them.